src/main/java/bio/terra/service/iam/LocalAuthenticatedUserRequestFactory.java

```diff
@@ -6,9 +6,11 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Profile;
 import org.springframework.stereotype.Component;
 
 @Component
+@Profile({"!terra", "!dev", "!integration"})
```

Comment:
Is that different than this?
```suggestion
@Profile({"google"})
```

Reply:
That works too...that's our only "local" spring profile right?

Reply:
Actually after looking at the code it's not clear to me if these are exclusive settings or not so maybe it's best to leave it using the `!` versions.

Reply:
Oh you're right, I think we use multiple profiles for some of our deployments...I'll double check the helm files

Reply:
These aren't mutually exclusive so I'll change that back!